signac/contrib/project.py

```diff
@@ -410,6 +410,9 @@ def _check_schema_compatibility(self):
     def min_len_unique_id(self):
         """Determine the minimum length required for a job id to be unique.
 
+        This operation's runtime scales with the number of jobs in the
```

Comment:
```suggestion
        This function's runtime scales with the number of jobs in the
```
I suggest reserving "operation" for functions related to signac-flow.

Reply:
Definitely, that's an oversight on my part. I'm going to use the word "method" because that's the usual name for a function attached to an object.